.catch(function(err) {
  ${1}
});
